config/example/config.toml

```
@@ -175,7 +175,7 @@ ipc_screens = [
 # Enable/disable the database on a whole
 sql_enabled = false
 # SQL version. If this is a mismatch, round start will be delayed
-sql_version = 56
+sql_version = 57
```

```suggestion
sql_version = 58
```
We have moved on to 58 now.